row_053_views/row_053_views_unsafe.py

```
@@ -0,0 +1,334 @@
+
+        data = RequestFile.objects.create(
+            fid = id,
+            requester=email,
+            uploader=email,
+            filename=file.filename,
+            file=decrypted_file_path,  # Save the new decrypted file path
+            rsa_encrypted_key=file.rsa_encrypted_key,
+            encrypted_data=aes_decrypted.encode(),
+            private_key=file.private_key,
+            status='Decrypted',
+            otp=random.randint(1111,9999)
```

Ruff (0.14.5)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)